src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -281,4 +281,11 @@ export default App.extend({
       lastSelectedIndex: null,
     });
   },
+  onStop() {
+    if (this.isRestarting()) {
+      this.getState().clearSearchQuery();
```

Comment:
After looking over this again I think `this.setState('searchQuery', '')` here would work just as well with a bit less code.

Reply:
Works for me, just pushed that change.